### ISSUE TITLE
Add warning about using NFS storage for RWO (Mongo)

### ISSUE THREAD

**Issue by BrendanFishback** [ibm-mas/installer#1266]
### Feature and motivation

There have been many issues in the past related to NFS storage in relation to Mongo during MAS installations*.

I would recommend either adding a warning within the interactive install itself, or adding it elsewhere in MAS CLI / Ansible documentation. A link to https://www.mongodb.com/docs/manual/administration/production-notes/#std-label-production-nfs would also be a good idea.

*If desired, I can provide a list of MAS support cases related to Mongo and NFS.

### Usage example

This addition would help lower the amount of issues (support cases) that arise or guide administrators to using other storage options instead.

**Comment by durera** [comment]
@BrendanFishback do we have any technotes published related to this that we can reference in the warning?

**Comment by ghost** [comment]
--> Tracking as part of serviceability submission

**Comment by BrendanFishback** [comment]
> @BrendanFishback do we have any technotes published related to this that we can reference in the warning?

We do not, I think mainly because the NFS concerns have been an 'unofficial' stance brought up through various cases. @terenceq has dealt with a few. 

I'll search through some of the past situations where NFS was an issue to see if any would work as technotes.

**Comment by ghost** [comment]
Just wanted to check if you still need this tracked with a serviceability request. If so, can someone provide me with the details of what your desired outcome is for this item? I can then take that info to PM to see if we can get this moving again. Thanks

**Comment by BrendanFishback** [comment]
@Mleftysup The desired outcome would be for our documentation and/or the MAS CLI prompts to warn users about using NFS with Mongo, with a link towards Mongo's own documentation on this topic: https://www.mongodb.com/docs/manual/administration/production-notes/#std-label-production-nfs

**Comment by durera** [comment]
I don't think it will be just mongo that has potential performance issues when nfs is uses as the RWO class.

I think the action here is a general - not specific to mongo - warning about the use of NFS as the RWO storage class ... this class is intended to be a local/block storage device generally speaking, expected for higher throughput usecases versus RWX where it's generally expected it will be remote/network storage.

We can put this information in the CLI at the storage class selection prompt, adaptng the language used in mongo's doc about this as I think they position it well:

> Note: Because remote file systems are often slower than local file systems, using a storage class backed by a remote file system (e.g. NFS) as the RWO storage class may result in degraded performance

**Comment by BrendanFishback** [comment]
Hi @durera 
This was brought up on another team call as we've seen more clients recently encountering issues due to NFS. Is there any way this warning could be expedited to be included in an upcoming MAS CLI release?

**Comment by durera** [comment]
<img width="2141" height="480" alt="Image" src="https://github.com/user-attachments/assets/f2d15e84-ccc8-4797-9a35-89c57d584c3e" />
